heka/files/toml/filter/lma_alarm.lua

```diff
@@ -0,0 +1,47 @@
+local M = {}
+setfenv(1, M) -- Remove external access to contain everything in the module
+
+local alarms = {
+{%- for _trigger_name in alarm.triggers %}
+{%- set _trigger = trigger.get(_trigger_name) %}
+{%- if _trigger is not none and _trigger.get('enabled', True) %}
```

Comment:
I checked and using `{% if _trigger %}` instead of `{% if _trigger is not none %}` actually works. `_trigger` does evaluate to `False` when `_trigger` is `None`.